client/scss/core.scss

```diff
@@ -162,6 +162,7 @@ These are classes for components.
 @import 'components/preview-panel';
 @import 'components/preview-error';
 @import 'components/form-side';
+@import 'components/userbar';
```

Comment:
Just to confirm: does this mean we're bringing the userbar styles into the compiled `core.css`, and thus we need to ensure that the styles in `_userbar.scss` is namespaced correctly (e.g. wrapped in `.w-userbar`-prefixed selectors, all seems fine currently) instead of just relying on Web Component's shadow DOM?

I might've missed this on our standups, but what was the reasoning behind this change? Is it so that we can easily reuse the styles from the design system?

Reply:
Yes. Userbar styles are largely as-is from before it was a Web Component, so are heavily scoped. 

There’s little value in having userbar styles available elsewhere – the point is to be able to reuse other components with the userbar without having to worry about them working differently due to using a different stylesheet.

For example, we’d be able to reuse styles in `_icons.scss`, or our different form elements (e.g. checkbox) to make upcoming accessibility checks configurable.

Reply:
That makes sense!